pybleau/app/plotting/tests/test_dataframe_plot_factories.py

```diff
@@ -396,8 +397,9 @@ def test_multi_line_2_curve_plot(self):
         self.assert_renderers_have_distinct_colors(plot)
 
     def test_multi_line_3_curve_plot(self):
-        renderer_styles = [LineRendererStyle(), LineRendererStyle(),
-                           LineRendererStyle()]
+        renderer_styles = [LineRendererStyle(color="blue"),
+                           LineRendererStyle(color="red"),
+                           LineRendererStyle(color="green")]
```

Comment:
Same comment: the point of this test was that you didn't have to specify different colors, they would be generated by the code. Did we lose that with this version?

Reply:
Same comment as above.

> Same comment: the point of this test was that you didn't have to specify different colors, they would be generated by the code. Did we lose that with this version?